website/addons/twofactor/static/twoFactorUserConfig.js

```diff
@@ -36,7 +36,9 @@ function ViewModel(qrCodeSelector, otpURL) {
         osfHelpers.postJSON(
             SETTINGS_URL,
             {code: self.tfaCode()}
-        ).done(function() {
+        ).done(function(response) {
+            $('#Tfa-success-message').html(response['message']);
+            $('.close').click();
```

Comment:
`push_status_message` in `framework/status/__init__.py`. If that had an additional parameter like `extra_classes` and would add those to `css_class` before it gets pushed to the list of statuses, then they would show up on the resultant page. Then you'll just need to find where the TFA model is calling `push_status_message` and pass the appropriate class through.

After all that, the class will be displayed in the frontend :)